lib/init-input.js

```diff
@@ -1,3 +1,5 @@
+/* jshint ignore: start */
```

Comment:
It seems this file isn't meant to be a valid javascript file. Notice how `prompt` is undefined, but this file still takes effect when you do `jpm init`. This is an input file for "promzard", see: https://github.com/npm/promzard#usage